js/pivottable/export_renderers.js

```diff
@@ -67,14 +75,73 @@
           r = result[n];
           text += r.join("\t") + "\n";
         }
-        return $("<textarea>").text(text).css({
-          width: ($(window).width() / 2) + "px",
-          height: ($(window).height() / 2) + "px"
+
+        return $('<a id="download" href="data:text/tsv,' + encodeURIComponent(text) + '"> Download as a TSV File </a>').click(function() {
+          $('#download').attr('download', 'pivot_report_' + getcurrenttimestamp() + '.tsv');
+        });
+     },
+
+    "CSV Export": function(pivotData, opts) {
+        var agg, colAttrs, colKey, colKeys, defaults, i, j, k, l, len, len1, len2, len3, len4, len5, m, n, r, result, row, rowAttr, rowAttrs, rowKey, rowKeys, text;
```

Comment:
This script is a library coming with pivottable.js library. See: https://github.com/nicolaskruchten/pivottable/blob/master/dist/export_renderers.js
It's just updated with functionality to download of generated TSV/CSV content.
But major part comes from original script (these short not readable variable names and code format).
So should I rewrite the original code?